stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -987,7 +1009,10 @@ public class ChatClient internal constructor(
         public val isInitialized: Boolean
             get() = instance != null
 
-        public fun isValidRemoteMessage(remoteMessage: RemoteMessage, defaultNotificationConfig: NotificationConfig = NotificationConfig()): Boolean =
+        public fun isValidRemoteMessage(
+            remoteMessage: RemoteMessage,
+            defaultNotificationConfig: NotificationConfig = NotificationConfig()
+        ): Boolean =
```

Comment:
This method should just have a proper body at this point, doesn't make sense for it to have an expression body